src/open_inwoner/js/components/confirmation/index.js

```diff
@@ -13,7 +13,10 @@ class Confirmation {
       const modalId = document.getElementById('modal')
       const modal = new Modal(modalId)
       modal.setTitle(this.form.dataset.confirmTitle)
-      modal.setText(this.form.dataset.confirmText)
+
+      if (modal.setText === true) {
+        modal.setText(this.form.dataset.confirmText)
+      }
```

Comment:
Hmm :thinking: I think you meant:

````js
if (this.form.dataset.confirmText) {
    modal.setText(this.form.dataset.confirmText)
}
// or maybe just
modal.setText((this.form.dataset.confirmText || ""))
````

Reply:
Oooooh @Bartvaderkin - the shorthand version looks neat!

Reply:
@jiromaykin note as javascript shorthands goes it has edge cases, like you could never print the number 0.

Reply:
@Bartvaderkin Thanks, I am assuming in case of strings we will never have zero, but I will remember this for future oddities.